_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: single
+title:  "Karlsruhe to D.C. ― a Dask story"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering Dask distributed
+header:
+  overlay_image: assets/images/dask_icon.svg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker & Nefta Kanilmaz
+author_profile: true
+---
+# Karlsruhe to D.C. ― a Dask story
+
+Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, we had not only connected with many of the core developers in the Dask community, but also had an (almost finished) implementation of a distributed semaphore in our bags. 
```

Comment:
Suggestions:
"... to attend the first ever Dask developer conference"
"... to Germany, ~~we had not only connected~~ not only had we connected..."
"... but ~~also had an (almost finished) implementation~~ had also (almost finished) an implementation of a distributed semaphore ~~in our bags.~~ [*unless* that's a `dask.bag` pun, in which case, please keep the original working]

```suggestion
Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first ever Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, not only had we connected with many of the core developers in the Dask community, but we had also (almost finished) an implementation of a distributed semaphore. 
```

Reply:
> unless that's a dask.bag pun, in which case, please keep the original working

I thought it kinda was (@nefta-kanilmaz-jdas ?), I'll keep that for now

```suggestion
Back in February 2020, we (Florian Jetter, Nefta Kanilmaz and Lucas Rademaker) travelled to the Washington D.C. metropolitan area to attend the first ever Dask developer conference. Our primary goal was to discuss Blue Yonder’s issues related to Dask with the attending developers and users. When we returned back to Germany, not only had we connected with many of the core developers in the Dask community, but we had also (almost finished) an implementation of a distributed semaphore in our bags. 
```